packages/core-data/src/entity-provider.js

```diff
@@ -74,16 +74,17 @@ export function useEntityId( kind, type ) {
  * specified property of the nearest provided
  * entity of the specified type.
  *
- * @param {string} kind The entity kind.
- * @param {string} type The entity type.
- * @param {string} prop The property name.
+ * @param {string} kind  The entity kind.
+ * @param {string} type  The entity type.
+ * @param {string} prop  The property name.
+ * @param {string} [_id] A fallback entity ID to use if no entity provider exists.
  *
  * @return {[*, Function]} A tuple where the first item is the
  *                          property value and the second is the
  *                          setter.
  */
-export function useEntityProp( kind, type, prop ) {
-	const id = useEntityId( kind, type );
+export function useEntityProp( kind, type, prop, _id ) {
+	const id = useEntityId( kind, type ) || _id;
```

Comment:
```suggestion
export function useEntityProp( kind, type, prop, id = useEntityId( kind, type ) ) {
```

Reply:
will this work with useEntityId being a hook?

Reply:
If it doesn't, you could probably do something like this instead:
```suggestion
export function useEntityProp( kind, type, prop, _id ) {
	const fallbackId = useEntityId( kind, type );
	const id = _id ?? fallbackId;
```

Reply:
yeah, that was my plan. :) I'll go with that because it feels safer, I guess. I know for sure that the linter won't allow you to do `const id = _id || useEntityId()`, and that feels like the same thing

Reply:
done 👍

Reply:
`id` doesn't change for the lifetime of the component, so it's safe to ignore eslint here.

But, I guess your approach is harmless.